app/javascript/src/components/Player.tsx

```diff
@@ -223,102 +225,118 @@ class Player extends React.Component<PlayerProps, PlayerState> {
   }
 
   private renderDebugger() {
```

Comment:
if a story has no stats or items configured, then exposing debug options would be superfluous and potentially confusing. Is there an easy way to check for the existence of those in a story before rendering any debugger UI?

Reply:
Yeah. There would be an easy way to check.